core/src/main/java/hudson/util/io/TarArchiver.java

```diff
@@ -33,6 +33,8 @@
 import java.io.OutputStream;
 import java.nio.file.Files;
 import java.nio.file.InvalidPathException;
+import java.nio.file.attribute.BasicFileAttributes;
+
```

Comment:
I think this blank line will trip up the linters. Better to run `mvn clean verify -DskipTests` locally before posting a PR to minimize turnaround time over stuff like this.

Reply:
I should not forget to disable the `quick-build` ;)